src/Altinn.App.Core/Models/UserAction/ActionError.cs

```diff
@@ -0,0 +1,27 @@
+using System.Text.Json.Serialization;
+
+namespace Altinn.App.Core.Models.UserAction;
+
+/// <summary>
+/// Defines an error object that should be returned if the action fails
+/// </summary>
+public class ActionError
+{
+    /// <summary>
+    /// Machine readable error code
+    /// </summary>
+    [JsonPropertyName("code")]
+    public string Code { get; set; } = string.Empty;
```

Comment:
Could this also be `required` instead of setting a default?

Reply:
I'm not sure, our frontend code would probably not read this field, but it is meant for those applications with api consumers. Should we force apps without API consumers to set a field that is never used?

Reply:
That is why I'm asking. If it is optional, it might make more sense to have it nullable?

Reply:
good point, lets set it as nullable 👍